ui/shared/components/form/IGVUploadField.jsx

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Table } from 'semantic-ui-react'
+
+import { NoBorderTable } from '../StyledComponents'
+import FileUploadField, { validateUploadedFile } from './XHRUploaderField'
+
+const IgvDropzoneLabel = ({ columns }) => (
+  <NoBorderTable basic="very" compact="very">
+    <Table.Body>
+      <Table.Row>
+        <Table.Cell colSpan={2}>
+          Upload a file that maps seqr Individuals to IGV file paths. Include one row per track.
+          <br />
+          For merged RNA tracks, include one row for coverage and one for junctions.
+        </Table.Cell>
+      </Table.Row>
+      <Table.Row><Table.Cell /></Table.Row>
+      <Table.Row>
+        <Table.HeaderCell>File Format:</Table.HeaderCell>
+        <Table.Cell>Tab-separated file (.tsv) or Excel spreadsheet (.xls)</Table.Cell>
+      </Table.Row>
+      <Table.Row><Table.Cell /></Table.Row>
+      {columns.map((column, i) => (
+        <Table.Row>
+          <Table.HeaderCell>{`Column ${i + 1}${i + 1 === columns.length ? ' (Optional)' : ''}:`}</Table.HeaderCell>
+          <Table.Cell>{column}</Table.Cell>
+        </Table.Row>
+      ))}
+    </Table.Body>
+  </NoBorderTable>
+)
+
+IgvDropzoneLabel.propTypes = {
+  columns: PropTypes.arrayOf(PropTypes.object),
+}
+
+// eslint-disable-next-line react-perf/jsx-no-new-array-as-prop
+const NO_PROJECT_COLUMNS = [
+  'Individual ID',
+  'IGV Track File Path',
+  'gCNV Sample ID, to identify the sample in the gCNV batch path. Not used for other track types',
+]
+
+// eslint-disable-next-line react-perf/jsx-no-new-array-as-prop
+const COLUMNS = ['Project', ...NO_PROJECT_COLUMNS]
+
+export const UPLOAD_IGV_FIELD = {
+  name: 'mappingFile',
+  validate: validateUploadedFile,
```

Comment:
From what I can tell, this validator isn't able to check the column number/contents.  Should we also update [the backend](https://github.com/broadinstitute/seqr/blob/fb35df0b867504b71012d7d358cd4411c928faf9/seqr/views/apis/igv_api.py#L30) so that we throw an error if the wrong number of columns is supplied?

Reply:
this field uses different URLs different that already properly handle their own validation

Reply:
gotcha, do you mean `receive_igv_table_handler` and `receive_bulk_igv_table_handler`, or even different URLs?

Reply:
even more different

Reply:
ok.. I'm gonna ask about this in person cause I'm a bit confused but not a blocker here.